### PR TITLE
I230 Updated Tournament and Venue Table Columns

### PR DESCRIPTION
**Pull request by rezbyte**
## Change Summary
- Added the 'Capacity' & 'Rounds' columns to the venue & tournament tables respectively 
- Refactored the table & added a slot to override the rendering of specified columns
- Fixed the table not scrolling on overflow

### Change Form
- [x] The pull request title has an issue number
- [x] The change works by "Smoke testing" or quick testing
- [ ] The change has tests
- [ ] The change has documentation

### CODE REVIEW

File: pages/admin/signup-requests.vue [hunk 1]
```diff
@@ -31,101 +31,89 @@ onUnmounted(() => {
...
+              }
+            "
+          />
+          <Button
+            button-text="Reject"
+            button-color="bg-light-red"
+            text-color="text-dark-red"
+            size="small"
+            @click="handleDelete(row)"
+          />
```
Comment (ChantelleKerr) [inline pages/admin/signup-requests.vue:111]: It would be nice if there was some margin between the two buttons. This table is definitely an improvement!

**Review by ChantelleKerr — APPROVED** [review]
Nice work! Once again making the codebase cleaner! :D

**Comment by ChantelleKerr** [comment]
Sorry I accidently approved but can you please add some margin to the button, then you can merge :) 
Thanks @rezbyte

File: pages/admin/signup-requests.vue (hunk 1 above)
Comment (rezbyte) [reply pages/admin/signup-requests.vue:111]: I've added a constant `1em` gap between them.
Let me know what you think. :slightly_smiling_face:

**Review by ChantelleKerr — APPROVED** [review]
Great job! Officially Approved

**Pull request merged**